lib/std/os/windows/bits.zig

```diff
@@ -65,6 +65,7 @@ pub const SIZE_T = usize;
 pub const TCHAR = if (UNICODE) WCHAR else u8;
 pub const UINT = c_uint;
 pub const ULONG_PTR = usize;
+pub const PULONG = *ULONG;
```

Comment:
Prefer to use `*ULONG` directly rather than `PULONG`

Reply:
Yeah, Zig has more "specific" ways of identifying pointers `*ULONG` vs `[*]ULONG`.  `PULONG` is more ambiguous.